Dockerfile

```diff
@@ -50,6 +50,14 @@ RUN apt-get update \
     python3-enchant \
   && locale-gen en_US.UTF-8
 
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+RUN GROUP_NAME=$(getent group $USER_GID | awk -F':' '{print $1}') \
+  && if [ -z $GROUP_NAME ]; then groupadd --gid $USER_GID dependabot ; \
+     else groupmod -n dependabot $GROUP_NAME ; fi \
```

Comment:
@thepwagner updated it to rename the group if it exists, seems to work in `docker-dev-shell`